class-parsely.php

```diff
@@ -0,0 +1,1813 @@
+<?php
+/**
+ * This is the main class for Parsely
+ *
+ * @category   Class
+ * @package    Parsely
+ */
```

Comment:
The file-level DocBlock, and the class-level DocBlock should be different things.

Reply:
See https://github.com/GaryJones/plugin-boilerplate/blob/master/src/Plugin.php as an example of the difference.

Reply:
Not sure I really follow... I think that means we need to drop the `@category` annotation...?